src/vos/evtree.c

```diff
@@ -969,6 +969,22 @@ evt_tcx_reset_trace(struct evt_context *tcx)
 	evt_tcx_set_trace(tcx, 0, tcx->tc_root->tr_node, 0, false);
 }
 
+static bool
+evt_tcx_reset(void *tcx)
+{
+	memset(tcx, 0, sizeof(struct evt_context));
```

Comment:
It looks like you could call evt_ent_array_init() at least here, as well as the ref count and the magic value.